addon/content/es-modules/thunderbird-compat.js

```diff
@@ -56,6 +77,67 @@ if (browser.i18n) {
   browser.i18n = i18n;
 }
 
+export const Services = window.Services || {};
```

Comment:
I'm not quite sure why you're stubbing this, since `Services.locale` is only used in `getLocaleDirection` for getting the direction, and `Services.locale` is only available in privileged code.

The closest we have for WebExtensions is `browser.i18n`

Reply:
If `Services` is not defined, there's code that errors when running in the browser. As far as I could tell, there's nothing in `i18n` that will give us the locale direction, so I figured I would stub a function that exists rather than create a new one. However, maybe a standalone `getLocaleDirection` with no deps should just be exported and used?

Reply:
I've just taken a look around, and I can't see the code you reference. I tried stubbing changing the additions to not use Services, and nothing else broke as far as I can tell.

Services is a privileged item, so maybe you had been trying it with one of the modules included or something?

So unless I'm missing something, a standalone `getLocaleDirection` would be fine - though I'm still debating if that's an API we should ask for in Thunderbird, or if we can just implement it ourselves.

Reply:
I just tried moving the `Services.locale` stuff into the dev frame, but I realized another issue, which is that there's currently no way to set the language from Conversations (which makes sense since it should be reading the language from Thunderbird). However, he dev frame wants to set the language and have those changes propogate through the UI.

Would you be okay with a function to change the language exported from `thunderbird-compat.js` that only gets used in the dev frame?

Reply:
That looks fine, thank you.